package-lock.json

```diff
@@ -0,0 +1,1417 @@
+{
```

Comment:
I appreciate adding a `package-lock.json`, but could I ask for this to be done in a separate PR? Thanks in advance.

Reply:
Actually, I've gitignored that file in #33. Feel free to rebase your branch and this file should disappear.